portal/src/tailwind.css

```diff
@@ -2,7 +2,7 @@
 
 /* Ensure radix styles overrides tailwind base styles */
 /* See https://www.radix-ui.com/themes/docs/overview/styling#tailwind-base-styles */
-@import "@radix-ui/themes/styles.css";
+/* @import "@radix-ui/themes/styles.css"; */
```

Comment:
Is this intended?

Reply:
Not intended, it was done for debugging. Removed